packages/xo-web/src/xo-app/logs/log-alert-body.js

```diff
@@ -174,7 +174,8 @@ export default [
                   8
                 )}) <TaskStateInfos status={taskLog.status} />{' '}
                 {scheduleId !== undefined &&
-                  taskLog.status === 'failure' && (
+                  (taskLog.status === 'failure' ||
+                    taskLog.status === 'interrupted') && (
```

Comment:
What about skipped status?
I think you should test the status that should not be retry-able.